lib/cache/cache.go

```diff
@@ -1014,10 +1015,26 @@ func (c *Cache) GetClusterConfig(opts ...services.MarshalOption) (services.Clust
 	return rg.clusterConfig.GetClusterConfig(services.AddOptions(opts, services.SkipValidation())...)
 }
 
+var (
+	clusterNameNotFound = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: teleport.MetricClusterNameNotFound,
+			Help: "Number of times a cluster name was not found",
+		},
+	)
+)
```

Comment:
So I think I was the one who originally suggested this go in `lib/cache`, but I can't for the life of me remember why I thought this.  Looking at it now, I think `lib/services/local` would be a more appropriate place 😳